test_scripts/unit_tests/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ubuntu:18.04
+RUN apt-get update && apt-get install -y python procps git && apt-get clean
+ADD https://github.com/bats-core/bats-core/archive/refs/tags/v1.3.0.tar.gz /bats/
+ADD https://github.com/ztombol/bats-support/archive/refs/tags/v0.3.0.tar.gz /bats/bats-support/
+ADD https://github.com/ztombol/bats-assert/archive/refs/tags/v0.3.0.tar.gz /bats/bats-assert/
+RUN tar -xzvf /bats/v1.3.0.tar.gz -C /bats/ && \
```

Comment:
Unrelated but large layer interactions like this can trigger A/V failures which are then hidden and you end up with weird generic timeouts or layer failures that make no sense. This is the right way to do it though, if you get a strange error in the future just check for A/V and/or system/filesystem logs to see if it failed to create the layer.

Reply:
Good to know